pupil-spa/src/environments/environment.ts

```diff
@@ -5,6 +5,8 @@
 
 export const environment = {
   apiURL: 'http://localhost:3001',
+  authURL: 'http://localhost:3001/api/questions',
+  // authURL: 'http://localhost:3003/auth',
```

Comment:
Can this commented-out line be deleted?

Reply:
its to make it obvious what to set it to when we enable the API endpoints